recipe/meta.yaml

```diff
@@ -37,8 +37,7 @@ requirements:
     - nose
     - pyparsing
     - pytz
-#    - py2cairo  # [linux and py2k]
```

Comment:
Why do we have this commented out? I guess we should remove it and open an issue to track this...